src/idiom270.fuzion

```diff
@@ -0,0 +1 @@
+if f64.is_NaN r1 then "This is a NaN" else "This is a number"
```

Comment:
This is not what the spec says: 
```
Given a floating point number r1 classify it as follows:
If it is a signaling NaN, print "This is a signaling NaN."
If it is a quiet NaN, print "This s a quiet NaN."
If it is not a NaN, print "This is a number."
```
Split code into several lignes.

Reply:
That's right, but if I understood it correctly, Fuzion does not differentiate between quiet and signalling NaNs and treats all NaNs as quiet.

Would it be better to change the output for the NaN case to `"This is a quiet NaN"`?

Reply:
@simonvonhackewitz You can create an issue for this in the main repo.